source/Playnite/GamesEditor.cs

```diff
@@ -467,6 +467,206 @@ public void OpenGameLocation(Game game)
             }
         }
 
+        public void CalculateGameSize(Game game, bool onlyIfDataMissing, bool updateGame = true, bool checkLastScanDate = false)

```

Comment:
I think this should be renamed to UpdateGameSize, since it doesn't just calculates and returns size, but also actually changes data in the library.